sklearn/utils/extmath.py

```diff
@@ -357,6 +361,166 @@ def randomized_svd(M, n_components, n_oversamples=10, n_iter='auto',
         return U[:, :n_components], s[:n_components], V[:n_components, :]
 
 
+def lobpcg_svd(M, n_components, n_oversamples=10, n_iter='auto',
```

Comment:
Should this be separately tested? Cases such as `flip_sign=False` seem to never be used in the current PR.

Reply:
I believe that this is separately extensively tested in sklearn\decomposition\tests\test_pca.py 
I am unsure about the use/need for `flip_sign=False`